frontend/dashboard/components/RepoList/RepoNameWithLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { getRepoEditUrl } from '../../utils/urlUtils';
+import { Link } from '@digdir/design-system-react';
+import classes from './RepoList.module.css';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+type RepoNameWithLinkProps = {
+  repoFullName: string;
+};
+
+export const RepoNameWithLink = ({ repoFullName }: RepoNameWithLinkProps): React.ReactElement => {
+  const { t } = useTranslation();
+
+  const [org, repoName] = repoFullName.split('/');
+  const isDatamodelling = repoFullName.endsWith('-datamodels');
```

Comment:
```suggestion
  const isDataModelling = repoFullName.endsWith('-datamodels');
```

Reply:
Is it more presice to use; `isDataModelRepo`?

Reply:
Yep! ✅